src/tests/issues.targets

```diff
@@ -2741,6 +2741,39 @@
         <ExcludeList Include = "$(XunitTestBinBase)/JIT/SIMD/VectorConvert_ro_Target_64Bit/**">
             <Issue>https://github.com/dotnet/runtime/issues/75359</Issue>
         </ExcludeList>
+        <ExcludeList Include = "$(XunitTestBinBase)/JIT/HardwareIntrinsics/X86/Aes/Aes_ro/**">
```

Comment:
do we want to disable for a particular config or all Mono runs ?

Reply:
This change disable those test only for llvmaot